src/diffusers/models/embeddings.py

```diff
@@ -600,3 +600,61 @@ def forward(self, boxes, masks, positive_embeddings):
 
         objs = self.linears(torch.cat([positive_embeddings, xyxy_embedding], dim=-1))
         return objs
+
+
+class PositionNetTextImage(nn.Module):
```

Comment:
We need to add a docstring here describing where this class comes from

Reply:
Actually instead of adding a new class here, can we put all the logic directly into `PositionNet` to remove duplicated logic? 

Could you add a new config variable `feature_type="text-only"  # "text-image"` here: https://github.com/huggingface/diffusers/blob/12358b986f15d0280f701034e42e9486253f2197/src/diffusers/models/embeddings.py#L566

It's important that we try to keep the `src/diffusers/models/embeddings` file as short as possible